pkg/gen/input/dependabot/internal/file/dependabot.go

```diff
@@ -38,12 +38,28 @@ updates:
   open-pull-requests-limit: 10
 {{- if .Reviewers }}
   reviewers:
-  {{- range $reviewer:= .Reviewers }} 
-  - {{ $reviewer }} 
+  {{- range $reviewer:= .Reviewers }}
+  - {{ $reviewer }}
   {{- end}}
 {{- end }}
   ignore:
   - dependency-name: k8s.io/*
     versions:
     - ">=0.17.0"
+- package-ecosystem: docker
+  directory: "/"
+  schedule:
+{{- if .Daily }}
+    interval: daily
```

Comment:
As a good practice I think the `--daily` flag should become `--interval` where allowed values are `daily` and `weekly` (default).

Reply:
Changed it to interval